apps/landingPage/package.json

```diff
@@ -1,37 +1,30 @@
 {
   "name": "@kiwicom/margarita-landing-page",
-  "private": true,
-  "version": "0.1.0",
+  "version": "0.0.0",
+  "license": "MIT",
+  "main": "index.js",
+  "scripts": {
+    "dev": "next",
+    "start": "yarn dev",
```

Comment:
Can we start landing page on different port, so it will not battle with margarita-web?
Basically like in `apps/documentation`, which is using `1337`. Just pick some different one.
(Can be in next PR)

Reply:
Sure, its on the 3001 now